frappe_docker have a issue with /ms-vscode.js-debug/bootloader.js

1. I try follow [this steps](https://github.com/frappe/frappe_docker/tree/develop/development ) but when i try `Execute Remote Containers : Reopen in Container` in console i receive:

```
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '/home/frappe/.vscode-server/data/User/workspaceStorage/c4e11f83fc3a369c2e61424104c5f10f/ms-vscode.js-debug/bootloader.js'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at Module._preloadModules (internal/modules/cjs/loader.js:1278:12)
    at loadPreloadModules (internal/bootstrap/pre_execution.js:439:5)
    at prepareMainThreadExecution (internal/bootstrap/pre_execution.js:71:3)
    at internal/main/run_main_module.js:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
nvm is not compatible with the npm config "prefix" option: currently set to ""
Run `nvm use --delete-prefix v12.18.2 --silent` to unset it.
frappe@c32d08f4f277:/workspace/development
```

if i try `bench --version` i receive:

```
frappe@c32d08f4f277:/workspace/development$ bench --version
bench is installed in editable mode!

This is not the recommended mode of installation for production. Instead, install the package from PyPI with: `pip install frappe-bench`

WARN: bench is installed in editable mode!

This is not the recommended mode of installation for production. Instead, install the package from PyPI with: `pip install frappe-bench`

Command not being executed in bench directory
WARN: Command not being executed in bench directory
5.1.0

```

but when try `bench init --skip-redis-config-generation --frappe-branch version-12 frappe-bench`


```
frappe@d62b0f27c180:/workspace/development$ bench init --skip-redis-config-generation --frappe-branch version-12 frappe-bench
bench is installed in editable mode!

This is not the recommended mode of installation for production. Instead, install the package from PyPI with: `pip install frappe-bench`

WARN: bench is installed in editable mode!

This is not the recommended mode of installation for production. Instead, install the package from PyPI with: `pip install frappe-bench`

$ /home/frappe/.local/bin/virtualenv -q env -p python3
Already using interpreter /usr/bin/python3

Getting frappe
$ git clone https://github.com/frappe/frappe.git --branch version-12 --depth 1 --origin upstream
Cloning into 'frappe'...
remote: Enumerating objects: 2639, done.
remote: Counting objects: 100% (2639/2639), done.
remote: Compressing objects: 100% (2453/2453), done.
remote: Total 2639 (delta 357), reused 1030 (delta 135), pack-reused 0
Receiving objects: 100% (2639/2639), 12.41 MiB | 1.14 MiB/s, done.
Resolving deltas: 100% (357/357), done.

Installing frappe
$ frappe-bench/env/bin/pip install -q -U -e frappe-bench/apps/frappe 
$ yarn install
[Errno 2] No such file or directory: 'yarn': 'yarn'
ERROR: There was a problem while creating frappe-bench
Do you want to rollback these changes? [Y/n]: 
```
I believe that the issue is:

```
frappe@d62b0f27c180:/workspace/development$ yarn --version
bash: yarn: command not found
```


I tried fix that adding this:

```
RUN install_packages \
    yarn \

```
in `/frappe_docker/build/bench/Dockerfile`

but no function it


please help me



thanks in advace,

Santiago

i tried again : 

```
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '/home/frappe/.vscode-server/data/User/workspaceStorage/c4e11f83fc3a369c2e61424104c5f10f/ms-vscode.js-debug/bootloader.js'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at Module._preloadModules (internal/modules/cjs/loader.js:1278:12)
    at loadPreloadModules (internal/bootstrap/pre_execution.js:439:5)
    at prepareMainThreadExecution (internal/bootstrap/pre_execution.js:71:3)
    at internal/main/run_main_module.js:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
nvm is not compatible with the npm config "prefix" option: currently set to ""
Run `nvm use --delete-prefix v12.18.2 --silent` to unset it.
```


i have download the docker images several times, i have tried delete all and clone the repository again and the result is the same.

i don't understand nothing :(

thanks for help.

--- a/development/.vscode/settings.json
+++ b/development/.vscode/settings.json
@@ -1,4 +1,4 @@
 {
     "python.pythonPath": "frappe-bench/env/bin/python",
-    "debug.node.autoAttach": "on"
+    "debug.node.autoAttach": "disabled"
 }

